.cargo/config

```diff
@@ -0,0 +1,7 @@
+[alias]
+run-wasm = "run --release --package run-wasm --"
```

Comment:
does this support `wasm32-unknown-unknown` only or the emscripten stuff as well?

Reply:
just wasm32-unknown-unknown.
The existing script only supports wasm32-unknown-unknown as well.

Reply:
#2233 could use essentially the same script but for emscripten target. I'm still unsure how much we really need emscripten support though.

Reply:
Alright, well let me know if I should implement emscripten support. I'm happy to do so if its needed by wgpu.